teuthology/suite/util.py

```diff
@@ -25,7 +25,7 @@
 
 log = logging.getLogger(__name__)
 
-CONTAINER_DISTRO = 'centos/8'       # the one to check for build_complete
+CONTAINER_DISTROS = ['centos/8', 'centos/9'] # the ones to check for build_complete
```

Comment:
```suggestion
CONTAINER_DISTROS = 'centos/9' # the ones to check for build_complete
```
This will error out once centos 8 builds start failing and/or are removed (since it will be checking that c8 AND c9 are completed). I think it is okay to move it strictly to centos 9 since we will lose centos 8 very soon.